src/migrations/1747997882050_add-communities.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions, PgType } from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('communities', {
+    id: {
+      type: PgType.UUID,
+      primaryKey: true,
+      notNull: true
+    },
+    name: {
+      type: PgType.VARCHAR,
+      notNull: true
+    },
+    description: {
+      type: PgType.TEXT,
+      notNull: true
+    },
+    owner_address: {
+      type: PgType.VARCHAR,
+      notNull: true
+    },
+    thumbnail_url: {
```

Comment:
we will have many thumbnails, right? I'm thinking about these options:
* make this an array (don't like it tbh)
* rename this to be `thumbnail_id` -- so then we can build static urls like: https://cdn.org/communities/thumbnails/<id>/256x256 or https://cdn.org/communities/thumbnails/<id>/512x512 and whatever sizes we do need there
* since we already have `id` we could use that to build static URLs and do not have any reference to thumbnails in the DB